### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by shyal** [cesarve77/simple-chat#7]
Hi cesarve77,

I'm just here to say thank you. This package is exemplary.

Shyal

**Comment by Netherdrake** [comment]
+1

Thank you very much for making this :)

**Comment by cesarve77** [comment]
Thanks guys

**Comment by openp2pdesign** [comment]
Agree, amazing package @cesarve77! I was thinking about integrating Rocket.chat in my Meteor app but the integration it's too complex, this package is perfect for me! Many thanks! :smiley:

I've created several issues and PRs, and while I wait I've created a forked package as `openp2pdesign:simple-chat`, the `master` branch in [my repo](https://github.com/openp2pdesign/simple-chat) is where I keep the sync with your version, the `mypackage` branch is for my package... let me know if this can help or if I have to change the setup!

**Comment by openp2pdesign** [comment]
@cesarve77 many thanks for adding me to the repo! I should have fixed several issues and moved to version 0.4.3, but I guess you have to do `meteor publish` for updating the package on AtmosphereJS...

**Comment by cesarve77** [comment]
Sure I will

**Comment by cesarve77** [comment]
done!


2018-03-09 12:41 GMT+10:00 Cesar Ramos <c@imagenproactiva.com>:
